File: app/src/main/java/com/android/ashwiask/tvmaze/db/TvMazeDatabase.java
package com.android.ashwiask.tvmaze.db;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.android.ashwiask.tvmaze.db.favouriteshow.FavoriteShow;
import com.android.ashwiask.tvmaze.db.favouriteshow.ShowDao;

@Database(entities = {FavoriteShow.class}, version = 1, exportSchema = false)
public abstract class TvMazeDatabase extends RoomDatabase {
    public static final String DATABASE_NAME = "tvmaze.db";

    public abstract ShowDao showDao();
}

File: app/src/main/java/com/android/ashwiask/tvmaze/db/TvMazeDatabase.kt
package com.android.ashwiask.tvmaze.db

import androidx.room.Database
import androidx.room.RoomDatabase
import com.android.ashwiask.tvmaze.db.favouriteshow.FavoriteShow
import com.android.ashwiask.tvmaze.db.favouriteshow.ShowDao

@Database(entities = [FavoriteShow::class], version = 2, exportSchema = false)
abstract class TvMazeDatabase : RoomDatabase() {

    abstract fun showDao(): ShowDao

    companion object {
        const val DATABASE_NAME = "tvmaze.db"
    }
}
